tools/FhirSchemaManager/InvokeAPI.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FhirSchemaManager.Exceptions;
+using FhirSchemaManager.Model;
+using Newtonsoft.Json;
+
+namespace FhirSchemaManager
+{
+    internal class InvokeAPI : IInvokeAPI
+    {
+        public async Task<List<CurrentVersion>> GetCurrentVersionInformation(Uri serverUri)
+        {
+            var httpClient = new HttpClient
```

Comment:
Would it make sense to do this in the constructor of the `InvokeAPI` and make it available through a private field? When you get into a process that calls multiple methods, it may make it easier.

Reply:
Makes sense, thanks!